runtime/onert/core/src/exec/ExecutorBase.cc

```diff
@@ -49,11 +49,8 @@ ExecutorBase::ExecutorBase(std::unique_ptr<ir::LoweredGraph> &&lowered_graph,
           tensor = tensor_registry->getNativeITensor(ind);
           if (tensor != nullptr)
           {
-            if (tensor_builder->supportDynamicTensor())
-            {
-              DynAllocInfo dyn_alloc_info{ind};
-              _input_to_dyn_alloc_info.emplace(tensor, dyn_alloc_info);
-            }
+            DynAllocInfo dyn_alloc_info{ind};
+            _input_to_dyn_alloc_info.emplace(tensor, dyn_alloc_info);
```

Comment:
This code looks confusing to me at this moment (since all input tensors enters their info as dynamic info)
but I will approve since you mentioned that `DynAllocInfo` will be removed later. :-)